src/renderers/shared/fiber/__tests__/ReactIncremental-test.js

```diff
@@ -20,7 +20,7 @@ describe('ReactIncremental', () => {
   beforeEach(() => {
     jest.resetModules();
     React = require('react');
-    ReactNoop = require('ReactNoop');
+    ReactNoop = require('react-noop-renderer');
```

Comment:
This seemed to make sense to me, I already used it for debugger in the past, and it would be nice to run tests on a Fiber bundle even for non-DOM specific tests.